tsdfileapi/rmq.py

```diff
@@ -36,8 +37,13 @@ def connect(self):
         self.connection = TornadoConnection(params)
         self.connection.add_on_open_callback(self.on_connect)
         self.connection.add_on_close_callback(self.on_closed)
+        self.connection.add_on_open_error_callback(self.on_open_error_callback)
+        self.connecting = False
         return
 
+    def on_open_error_callback(self, connection: TornadoConnection, exception: Exception) -> None:
+        logging.info('could not connect')
```

Comment:
Being an error, perhaps this should be changed to:
```suggestion
        logging.error('could not connect')
```

Reply:
good idea, done.